README.md

```diff
@@ -30,281 +30,19 @@
-    },
-)
-
-// Or use a single CREATE with content variable
-createResult, err := surrealdb.Query[[]Person](
-    context.Background(),
-    db,
-    "CREATE person:$id CONTENT $content",
-    map[string]any{
-        "id": "alice",
-        "content": map[string]any{
-            "name": "Alice",
-            "age": 28,
-            "city": "New York",
-        },
-    },
-)
-```
-
-The `Query` function supports:
-- Full SurrealQL syntax including transactions
-- Parameterized queries for security
-- Typed results with generics
-- Multiple statements in a single call
-
-### Using Send for low-level control
-
-All data manipulation methods are handled by an underlying `send` function. This function is
-exposed via `db.Send` function if you want to create requests yourself but limited to a selected set of methods. These
-methods are:
-
--   select
--   create
--   insert
--   upsert
--   update
--   patch
--   delete
--   query
-
-```go
-type UserSelectResult struct {
-	Result []Users
-}
-
-var res UserSelectResult
-// or var res surrealdb.Result[[]Users]
-
-err := db.Send(context.Background(), &res, "select", user.ID)
-if err != nil {
-	panic(err)
-}
-```
-
-## Connection Engines
-
-There are 2 different connection engines you can use to connect to SurrealDb backend. You can do so via Websocket or through HTTP
-connections
-
-### Via Websocket
-
-```go
-db, err := surrealdb.FromEndpointURLString(ctx, "ws://localhost:8000")
-```
-
-or for a secure connection
-
-```go
-db, err := surrealdb.FromEndpointURLString(ctx, "wss://localhost:8000")
-```
-
-### Via HTTP
-
-There are some functions that are not available on RPC when using HTTP but on Websocket. All these except
-the "live" endpoint are effectively implemented in the HTTP library and provides the same result as though
-it is natively available on HTTP. While using the HTTP connection engine, note that live queries will still
-use a websocket connection if the backend supports it
-
-```go
-db, err := surrealdb.FromEndpointURLString(ctx, "http://localhost:8000")
-```
-
-or for a secure connection
-
-```go
-db, err := surrealdb.FromEndpointURLString(ctx, "https://localhost:8000")
-```
-
-### Using SurrealKV and Memory
-
-SurrealKV and Memory also do not support live notifications at this time. This would be updated in the next
-release.
-
-> ⚠️ **Note**
-> Although the examples below reference `surrealkv://` and `memory://`, **these modes are currently not supported in the Go SDK**.
-> Support for embedded databases (like SurrealKV and in-memory) is pending future development and is being tracked in [issue #197](https://github.com/surrealdb/surrealdb.go/issues/197).
->
-> Until then, please use `http://` or `ws://` endpoints to connect to a running SurrealDB server instance.
-
-For Surreal KV
-
-```go
-db, err := surrealdb.New("surrealkv://path/to/dbfile.kv")
-```
-
-For Memory
-
-```go
-db, err := surrealdb.New("mem://")
-db, err := surrealdb.New("memory://")
-```
-
-## Data Models
```

Comment:
Moved to the `models` package README, so that it appears in https://pkg.go.dev/github.com/surrealdb/surrealdb.go/pkg/models, making the documentation more structured and therefore easier to consume.